test/unit/object/test_legal_hold_policy.py

```diff
@@ -119,20 +119,32 @@ def test_delete_policy_return_the_correct_response(
     assert response is True
 
 
+@pytest.mark.parametrize('assign_to_type, assign_to_id, params', [
+    (None, None, {}),
+    ('file', None, {'assign_to_type': 'file'}),
+    ('folder', '22222', {'assign_to_type': 'folder', 'assign_to_id': '22222'})
```

Comment:
In all 3 cases, params is duplicate information. Is it worth having it as a separate parameter to the test?